provider/oauth2/tests.py

```diff
@@ -634,9 +639,10 @@ def assert_valid_access_token_response(self, access_token, response):
     def get_latest_access_token(self):
         return AccessToken.objects.filter(client=self.get_client()).order_by('-id')[0]
 
-    def test_authorize_success(self):
+    @ddt.data(None, 'read')
```

Comment:
Can you use `constants.READ` here instead?
Also, didn't these 2 tests (`read` and `None`) work even before this PR?